application/src/main/java/org/togetherjava/tjbot/commands/help/AskCommand.java

```diff
@@ -173,4 +248,9 @@ private static void handleFailure(Throwable exception, InteractionHook eventHook
 
         logger.error("Attempted to create a help thread, but failed", exception);
     }
+
+    private static RestAction<String> mentionHelpChangeCommand(Guild guild, String subcommand) {
+        return MessageUtils.mentionGuildSlashCommand(guild, COMMAND_NAME, CHANGE_SUBCOMMAND_GROUP,
+                subcommand);
+    }
```

Comment:
Oops. This is probably supposed to be returning a mention of `/help-thread change category/title`. It currently returns `/ask change category/title`. This is probably because of you importing all static things of `HelpThreadCommand` and forgot that this class has the same field but with a different value.